src/services/logger.ts

```diff
@@ -90,10 +91,13 @@ export function debugObj(e: any) {
 // TODO is there a way to DRY that?
 // returns the sentry error id
 export function logErrSentry(e: any, uid?: string, email?: string, dbAlias?: string): string {
+  if (!(e instanceof DepError) && !(e instanceof Error)) {
+    singleton.error(`Invalid error type ${typeof e} when logging to sentry`)
+  }
```

Comment:
If we fail if is not `DepError` or `Error` why don't we:
```suggestion
export function logErrSentry(e: Error | DepError, uid?: string, email?: string, dbAlias?: string): string {
```

Reply:
Or if we want to specifically caught this kind of errors we could show the content of the error to not lose information:
```suggestion
export function logErrSentry(e: any, uid?: string, email?: string, dbAlias?: string): string {
  if (!(e instanceof DepError) && !(e instanceof Error)) {
    singleton.error(`Invalid error type ${typeof e} when logging to sentry. e = ${JSON.stringify(e)}`)
  }
```

Reply:
Catch statements are of unknown type unfortunately so we would have to cast it at the call site every time we call this function. Instead I am checking the type here.

Reply:
oh thats true!! well maybe my second suggestion can give us some extra idea of what happened if we send the wrong thing at some point?